nixos/modules/services/networking/bird-lg.nix

```diff
@@ -0,0 +1,260 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.bird-lg;
+in
+{
+  options = {
+    services.bird-lg = {
+      package = mkOption {
+        type = types.package;
+        default = pkgs.bird-lg;
+        defaultText = literalExpression "pkgs.bird-lg";
+        description = "The Bird Looking Glass package to use.";
+      };
+
+      user = mkOption {
```

Comment:
How is this user being created?

Reply:
This user should be the one used by bird, defined [here](https://github.com/NixOS/nixpkgs/blob/61ed7d351afd59164b5a224b640ee44d614a7fc0/nixos/modules/services/networking/bird.nix#L93-L100)

Reply:
But you don't know that this service is enabled. If this service has a hard requirement on another service an assertion should be added to this module.

Reply:
Would there be a security benefit from running these services as a separate user from the bird service?